### ISSUE TITLE
possibility for xoom.com

### ISSUE THREAD

**Issue by yfdyh000** [liberapay/salon#513]
https://www.xoom.com/
https://en.wikipedia.org/wiki/Xoom_Corporation
This is a direct money transfer service, under the PayPal umbrella.

I stumbled across it today and think this has the potential to be a receive method. I didn't find a relevant discussion in this project.

As far as I'm concerned, Stripe is still not available in China. PayPal is working after some steps -- it permanently froze my account after the first charge - against the terms. I had to proactively try to contact support, and then provide ID to unblock (and I am lucky). What's more, PayPal withdrawals will have high fees in some countries, or go through some third-party methods.

From my basic observations, the Xoom service allows direct, instant money transfers to bank accounts in many countries, with similar rates to PayPal. It also supports charging to mobile phone account. In this way, the recipient may avoid withdrawal fees from PayPal. However, I also noticed that the service may not provide the interface APIs, and there may be problems with refunds & taxes from receiver.

More about the interface & regions:
https://help.xoom.com/s/article/can-i-use-xoom-for-commercial-purposes "Xoom is a person-to-person remittance service and cannot be used for commercial purposes."   [_sad_]
https://help.xoom.com/s/article/xoom-basics
https://help.xoom.com/s/article/how-can-i-pay-with-my-paypal-account

**Comment by Changaco** [comment]
Xoom and its competitors are mostly for people who send money to the countries they emigrated from because they still have family there. These remittance services aren't designed for integration into a platform like Liberapay.

Closing in favour of <https://github.com/liberapay/liberapay.com/issues/1394>.
